book/chapters/flags.md

```diff
@@ -120,4 +120,4 @@ Flag int: 3
 Flag plain: 1
 ```
 
-[^1]: It will not inherit this from the parent defaults, since this is often useful even if you don't want all options to allow multiple passed options.
+\[^1\]: It will not inherit this from the parent defaults, since this is often useful even if you don't want all options to allow multiple passed options.
```

Comment:
Guessing this was required for some reason?

Reply:
In some parsers it interprets that as a link and give syntax warnings